OpenSourcePolitics/decidim-cd44
# Fix edit initiative form

#### Description

Edit initiative form is not saved when there is only on initiative type

Error on form validation
``` 
 @messages={:type_id=>["doit être rempli(e)"]}, @details={:type_id=>[{:error=>:blank}]}>
```

#### Testing

1. Seed application
2. Create initiative
3. Edit Initiative
4. Then remove all initiative type except one
5. Edit again the initiative
6. It should be updated
